### DISCUSSION TITLE
Inquiry Regarding Motion Data Issue in Pandora Engine 2.4 (2.3+)

### DISCUSSION THREAD

**Discussion by Rokypukes** [Monitor221hz/Pandora-Behaviour-Engine-Plus#335] [Q&A] [answered]
I hope this message finds you well. I have a question regarding the Pandora Engine that might seem quite basic, and I apologize in advance if this is indeed a trivial inquiry.

I reside in a country where English is not the primary language, and many users here have experienced the same issue following the update to version 2.3. We are unable to understand why it is not working as expected, and after attempting to troubleshoot within our community, we decided it would be best to reach out to you directly.

This issue specifically appears with Pandora version 2.15 and later. When running Pandora 2.15, the engine logs a message that reads, "40716 total animations added" upon completion. However, with Pandora 2.4, using the exact same mod list but simply switching the engine version, only around 12,000 animations are registered. This results in a discrepancy of roughly 30,000 animations compared to version 2.15.

In version 2.15, all animations worked without requiring FNIS, so we had been relying solely on the Pandora Engine. However, now, certain segments seem to be marked as "Skipped" during the engine’s operation. Could you advise us on how to restore the missing 30,000 animations? Should we run FNIS or Nemesis beforehand to prevent this issue?

The "Quickstart / MO2 Users" section of the documentation states the following:

"Run the program from within MO2, using the menu on the right."

We apologize if this is an oversimplified question and if we may have misunderstood the intended process. We are truly grateful for your hard work and support and are always cheering on your project.

Thank you very much.

ps. 
Please note that I used a translation tool for this message, and I apologize if any parts are unclear or difficult to understand as a result. Thank you very much for your patience and understanding.

**Comment by Monitor221hz** [comment, marked as answer]
Hello @Rokypukes ,

In 2.3.6-beta of the engine and later, it switches to using the registry path for Skyrim by default for Windows Users. Usually this defaults to the Steam install path of Skyrim. 

If your MO2 is managing a different game location for mods, you need to specify this using the `-tesv` startup argument in the executable settings screen of MO2. 

For example:

```-tesv:"D:\SteamLibrary\steamapps\common\Skyrim Special Edition"```

or if you're using a wabbajack install, something like:

```-tesv:"D:\Games\Test Modlist\Game Root"```

Please note there cannot be a space between `-tesv:` and the filepath enclosed in quotes (`-tesv: "D:\Games\Test Modlist\Game Root"` is wrong).

You can check the managed game path in the MO2 settings menu if you don't know what path to set `-tesv` to. 

I hope this comes out of the translator making sense. Let me know if you have any more questions.

**Reply by Rokypukes** [reply]
Thank you very much for your detailed explanation. I would like to organize your instructions step-by-step and confirm my understanding before sharing them with my community here.

Go to "Modify Executables" at the top right of MO2.

In the [Binary] section, specify Pandora Behaviour Engine+.exe and its path.

For the [Argument] section, you mentioned using a value like -tesv. I understood this as specifying the folder where my SkyrimSE.exe is located. For example, if my Skyrim is installed on my C drive, I would enter the following in [Argument]:

-tesv:"C:\Program Files (x86)\Steam\steamapps\common\Skyrim Special Edition"

If I have misunderstood any part of your instructions, please feel free to correct me.

After applying these settings, I launched the Pandora Behaviour Engine.
As a result, version 2.4 successfully loaded the animations. I am deeply grateful for your kind guidance and once again send my sincere thanks.
